apps/web/package.json

```diff
@@ -30,7 +36,7 @@
     "date-fns": "^2.30.0",
     "lucide-react": "^0.244.0",
     "micro": "10.0.1",
-    "next": "13.4.6",
+    "next": "13.4.8",
```

Comment:
Solves some issues I had locally with router interception. 🚀

Reply:
Also seems to improve compilation time 🚀